rotkehlchen/chain/evm/decoding/curve/decoder.py

```diff
@@ -300,31 +332,64 @@ def _decode_curve_deposit_events(
                 event.counterparty == CPT_CURVE
             ):
                 receive_event = event
+                received_asset = event.asset
+
+        if received_asset is None:  # find it through the logs
+            deposit_addresses = {event.address for event in deposit_events} | {ZERO_ADDRESS}
+            for _log in all_logs[all_logs.index(tx_log) + 1:]:
+                if (
+                    (_log.topics[0] == ERC20_OR_ERC721_TRANSFER) and
+                    (hex_or_bytes_to_address(_log.topics[1]) in deposit_addresses) and
+                    self.base.is_tracked(hex_or_bytes_to_address(_log.topics[2]))
+                ):
+                    received_asset = Asset(evm_address_to_identifier(
```

Comment:
you should use `self.base.get_or_create_evm_token()`. Never initialize a token like this from external data